std/node/fs.ts

```diff
@@ -0,0 +1 @@
+export { readFile, readFileSync } from "./fs/read_file.ts";
```

Comment:
Maybe just merge read_file.ts into this one - it will make for better docs on Deno.land and also I prefer to keep the number of files small until it becomes unwieldy to have so many functions in a single file - at that point we can break it out. So just fs.ts and fs_test.ts please.

Reply:
it was just a direct port but i get your point.